go/libraries/doltcore/sqle/schema_table.go

```diff
@@ -200,6 +216,55 @@ func migrateOldSchemasTableToNew(
 	return root, rowsToAdd, nil
 }
 
+func migrateLessOldSchemasTableToNew(
```

Comment:
Don't have two migration functions. Just write one that handles both older formats